static/app/components/events/interfaces/performance/resources.tsx

```diff
@@ -21,21 +21,13 @@ export function Resources(props: Props) {
   return (
     <EventDataSection type="resources-and-whatever" title={t('Resources and Whatever')}>
```

Comment:
```suggestion
    <EventDataSection type="resources" title={t('Resources')}>
```

now's the time?

Reply:
This was an intentional design decision, quirkiness is good. 😎

Reply:
(we can double check with Dora)

Reply:
😟